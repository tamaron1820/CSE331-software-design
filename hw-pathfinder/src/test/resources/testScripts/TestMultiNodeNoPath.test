# Testing Dijkstra's algorithm when there are no paths between multiple nodes

CreateGraph graph1

AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

FindPath graph1 n1 n2

FindPath graph1 n2 n3

FindPath graph1 n1 n3