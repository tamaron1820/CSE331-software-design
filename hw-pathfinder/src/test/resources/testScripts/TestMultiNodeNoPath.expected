# Testing Dijkstra's algorithm when there are no paths between multiple nodes

created graph graph1

added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

path from n1 to n2:
no path found

path from n2 to n3:
no path found

path from n1 to n3:
no path found